must-gather/collection-scripts/gather

```diff
@@ -32,19 +32,20 @@ mkdir -p ${BASE_COLLECTION_PATH}/oc_output/
 commands=()
 
 # collect oc output of OC commands
-commands+=("version")
```

Comment:
These changes aren't related to the new template, right? Seems like we should revert these changes.

Reply:
Indeed, there seems to be a wird openshift-bot merge commit https://github.com/openshift/ocs-operator/pull/481/commits/5bb89d20ad5a8092582adb8ebea182fca8ad691d called "aa" from Mar 05 in this PR.